requirements.txt

```diff
@@ -67,9 +67,7 @@ django-cors-headers==2.1.0
 djangorestframework-bulk==0.2.1
 hashids==1.2.0
 pyjwt==1.5.3
-# django-celery-beat==1.0.1  # BSD 3 Clause
-# Contains a fix for handling disabled tasks that still has not been release
-git+git://github.com/celery/django-celery-beat@f014edcb954c707cb7628f4416257b6a58689523# BSD 3 Clause
+django-celery-beat==1.2.0  # BSD 3 Clause
```

Comment:
👍 . This looks like a safe upgrade. Verified that 1.2.0 includes the patch that we need.